substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/c/CAnnotationProcessorCache.java

```diff
@@ -76,9 +76,16 @@
 
         @Option(help = "Exit image generation after C Annotation Processor Cache creation.")//
         public static final HostedOptionKey<Boolean> ExitAfterCAPCache = new HostedOptionKey<>(false);
+
+        @Option(help = "Output query code for target platform without executing it")//
+        public static final HostedOptionKey<Boolean> DontRunCAPQueryCode = new HostedOptionKey<>(false);
+
+        @Option(help = "Directory where query code for target platform should be output")//
+        public static final HostedOptionKey<String> CAPQueryCodeDir = new HostedOptionKey<>("");
```

Comment:
I'd call this `QueryCodeDir` to better differentiate it from the regular `CAP....` usecases (cached NativeCodeInfo result files).

Reply:
Changed